apps/fhir/bluebutton/migrations/0001_initial.py

```diff
@@ -12,7 +12,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('server', '0017_resourcerouter_wait_time'),
+        # ('server', '0017_resourcerouter_wait_time'),
```

Comment:
Can you help me understand why this is commented out? If you intend to leave this commented, could you add a code comment with an explanation?

Reply:
the resourcerouter has been removed (never used and obsolete), hence removed the dependency, should have delete the line instead of comment out, thanks JZ for reviewing this...